backend/app/models/publication.rb

```diff
@@ -35,11 +35,13 @@ class Publication < Content
   scope :older_pubs, -> { where('EXTRACT(year from content_date) < ?', Date.today.year - 5)}
   scope :filter_or_older_pubs, ->(years) {where('EXTRACT(year from content_date) IN (?) OR EXTRACT(year from content_date) < ?', years, Date.today.year-5)}
   scope :by_years, ->(years) { where('EXTRACT(year from content_date) IN (?)', years) }
+  scope :by_partners, ->(partners) { joins(:content_partners).where(content_partners: {id: partners})}
```

Comment:
if we are using it both in activities and publications we can move the scope to content.rb , as they both inherit from it.